anonchat-server/bin/anonchat_server.dart

```diff
@@ -13,10 +13,19 @@ Future<void> main(List<String> arguments) async {
 
   /// TCP server itself
   final server = await ServerSocket.bind('0.0.0.0', int.parse(arguments[0]));
-  print('\/\/ Launched server on port ${int.parse(arguments[0])}\n');
+  print('// Launched server on port ${int.parse(arguments[0])}\n');
   server.listen((socket) {
     // Add fresh connected client to the list
     sockets.add(socket);
+    // Send MOTD
+    bool MOTDtoggle = false;
```

Comment:
could be done, but idk how to work with flags yet

Reply:
actually this could be done, but we need this library https://pub.dev/packages/args

Reply:
Yea, good idea. I might try working with this package myself if I have some free time.

> actually this could be done, but we need this library https://pub.dev/packages/args

Reply:
But, i think its better to implement json file config for server

Reply:
> json file config for server

i agree, but that's a task for later